jujugui/static/gui/src/app/components/deployment-flow/test-deployment-flow.js

```diff
@@ -380,7 +381,24 @@ describe('DeploymentFlow', function() {
       modelCommitted: true
     });
     const output = renderer.getRenderOutput();
-    assert.isFalse(output.props.children[5].props.disabled);
+    assert.strictEqual(output.props.children[6].props.disabled, false);
+  });
+
+  it('displays the VPC section in new AWS models', function() {
+    const renderer = createDeploymentFlow({modelCommitted: false});
+    const instance = renderer.getMountedInstance();
+    instance._setCloud({name: 'aws'});
+    const output = renderer.getRenderOutput();
+    const expectedOutput = (
+      <juju.components.DeploymentSection
+        completed={false}
+        disabled={false}
+        instance="deployment-vpc"
+        showCheck={false}>
+        <juju.components.DeploymentVPC setVPCId={instance._setVPCId} />
+      </juju.components.DeploymentSection>
+    );
+    assert.deepEqual(output.props.children[4], expectedOutput);
```

Comment:
The `expect` branch has landed now so this should probably be converted to that.